tests/unit/main_html5/wrapper-event-tests.js

```diff
@@ -191,6 +191,31 @@ describe('main_html5 wrapper tests', function () {
     expect(vtc.notifyParameters).to.eql([vtc.interface.EVENTS.SEEKING]);
   });
 
+  it('should not raise seeking before initial time has seeked', function(){
+    vtc.interface.EVENTS.SEEKING = "seeking";
+    element.duration = 20;
+    spyOn(element.seekable, "start").andReturn(0);
+    spyOn(element.seekable, "end").andReturn(20);
+    element.seekable.length = 1;
+    wrapper.setInitialTime(10);
+    $(element).triggerHandler("seeking");
+    expect(vtc.notifyParameters[0]).to.not.eql(vtc.interface.EVENTS.SEEKING);
+    $(element).triggerHandler("seeked");
+    $(element).triggerHandler("seeking");
+    expect(vtc.notifyParameters[0]).to.eql(vtc.interface.EVENTS.SEEKING);
+  });
+
+  it('should raise seeking before initial time has seeked if initialtime is 0', function(){
```

Comment:
Oh, I didn't notice that pervious test has initial time set to 10, and this one to 0.  Never mind